source/MaterialXTest/GenShaderUtil.cpp

```diff
@@ -650,53 +650,59 @@ void ShaderGeneratorTester::validate(const mx::GenOptions& generateOptions, cons
                 nodeDef = shaderRef->getNodeDef();
             }
 
-            // Allow to skip nodedefs to test if specified
-            const std::string nodeDefName = nodeDef->getName();
-            if (_skipNodeDefs.count(nodeDefName))
+            if (!nodeDef)
             {
-                _logFile << ">> Skipped testing nodedef: " << nodeDefName << std::endl;
-                continue;
+                CHECK(nodeDef);
```

Comment:
This is a minor change. Just makes this code easier to debug by enabling users to place a breakpoint inside this if statement for when nodeDef is a nullptr. I found it useful to do so for debugging purposes so figured I'd contribute it back.